### PR TITLE
1.6: Ansible install instructions

### PR DESCRIPTION
**Pull request by Hwesta**
Added instructions for installing Archivematica from source in a production configuration via Ansible.  Formatting is semantic linebreaks (at the end of each sentence) but that can be changed.

A few questions:
* Should the quickstart be updated to install 1.6 instead of qa/1.x?
* Should the quickstart be updated to do a production deploy instead of a development one?
* Or, should the deploy-pub repo be updated to default to 1.6 and/or production? See artefactual/deploy-pub#25

~~A few style tweaks:~~
* ~~Removed leading whitespace that was causing an accidental block format~~
* ~~Removed newlines from lists in quickstart, since that was breaking formatting~~

Edit: since artefactual/deploy-pub#25 was merged, removed the two instructions modifying files in deploy-pub.  Also manually wrapped lines, based on discussions.

### CODE REVIEW

File: admin-manual/installation/installation.rst [hunk 1]
```diff
@@ -294,9 +294,84 @@ If you have trouble with the gearman command try this as an alternative:
...
+
+#. Set it to install Archivematica 1.6.  Modify the file `singlenode.yml` to say:
+
+   ..code:: yaml
+
+       - include_vars: "vars-singlenode-1.6.yml"
+
+#. Set it to install in production instead of development mode. Modify the file `vars-singlenode.yml` and **remove** the following line:
+
+   ..code:: ruby
```
Comment (Hwesta) [inline admin-manual/installation/installation.rst]: This and `..code:: yaml` don't seem to format correctly. This doesn't matter if they're removed (see other comment)

File: admin-manual/installation/installation.rst [hunk 2]
```diff
@@ -294,9 +294,84 @@ If you have trouble with the gearman command try this as an alternative:
...
+      cd deploy-pub/playbooks/archivematica
+      ansible-galaxy install -f -p roles/ -r requirements.yml
+
+#. Set it to install Archivematica 1.6.  Modify the file `singlenode.yml` to say:
+
+   ..code:: yaml
+
+       - include_vars: "vars-singlenode-1.6.yml"
+
+#. Set it to install in production instead of development mode. Modify the file `vars-singlenode.yml` and **remove** the following line:
```
Comment (Hwesta) [inline admin-manual/installation/installation.rst]: This and the previous instruction are not needed if artefactual/deploy-pub#25 is merged.

File: admin-manual/installation/installation.rst (hunk 1 above)
Comment (sevein) [reply admin-manual/installation/installation.rst]: The lexer [exists](http://pygments.org/docs/lexers/#pygments.lexers.data.YamlLexer), maybe we have an old version of pygments?

**Comment by sevein** [comment]
I'm sad that we're not wrapping text in archivematica-docs like we've been doing in atom-docs and many big projects out there do too (e.g. https://github.com/django/django/blob/master/docs/index.txt). 😭

**Comment by sevein** [comment]
Should we mention somewhere that Vagrant/VirtualBox are only for local testing/dev purposes? In AtoM we use rst admonitions for this kind of warning.

**Comment by Hwesta** [comment]
Hmm, it looks like if the wrapping in lists is also manually indented, the formatting works (judging by Django's example), so we could update that commit if we wanted to do manual wrapping instead of semantic line breaks.

File: admin-manual/installation/installation.rst (hunk 1 above)
Comment (Hwesta) [reply admin-manual/installation/installation.rst]: Possibly! I was only testing the formatting locally, I can't actually confirm if the production version has it. I'd rather not need these instructions if deploy-pub is updated. :)

**Comment by Hwesta** [comment]
> Should we mention somewhere that Vagrant/VirtualBox are only for local testing/dev purposes? In AtoM we use rst admonitions for this kind of warning.

I'm not sure that's true - I think we might support production vagrant installs.  @jhsimpson ?

**Comment by Hwesta** [comment]
Re: Text wrapping, it looks like Kelly fixed the formatting issues I found, so I'm going to rebase this without the formatting changes.

**Comment by jhsimpson** [comment]
VirtualBox is not for production use, at least it would be very much an exception.  Vagrant can work with vmware and other providers, but they way we have set up the deploy-pub repo, to use Vagrant with VirtualBox, would have to be modified, so I think it makes sense to have at least a mention if not a warning that this is not intended for production use.  If for no other reason than the networking would have to change, you can't get to the Vagrant install from outside your local machine without making changes.

File: admin-manual/installation/installation.rst [hunk 3]
```diff
@@ -294,9 +294,82 @@ If you have trouble with the gearman command try this as an alternative:
 Install from source
 -------------------
 
-Installing from source has been tested using ansible scripts. Ansible installations have been tested for new installations but are not fully tested for upgrades.
+Installing from source has been tested using ansible scripts. Ansible 
+installations have been tested for new installations but are not fully tested 
+for upgrades.
+
+While installing using Vagrant is well suited to trying out Archivematica, it is 
+not suitable for production deployment.
```
Comment (jhsimpson) [inline admin-manual/installation/installation.rst]: suggest changing:
While installing using Vagrant is well suited to trying out Archivematica, it is not suitable for production deployment

to:
These instructions are designed to create a test environment on your local machine.  The ansible roles referenced here can be used in production deployments by creating your own ansible playbook to run them.  See https://github.com/artefactual/deploy-pub/playbooks/archivematica for more details.

**Comment by jhsimpson** [comment]
:+1:

**Review by jhsimpson — APPROVED** [review]

**Pull request merged**